spdmlib/src/message/mod.rs

```diff
@@ -614,10 +614,7 @@ mod tests {
             payload: SpdmMessagePayload::SpdmVersionResponse(SpdmVersionResponsePayload {
                 version_number_entry_count: 0x02,
                 versions: gen_array_clone(
-                    SpdmVersionStruct {
-                        update: 100,
-                        version: SpdmVersion::SpdmVersion11,
-                    },
+                    SpdmVersionStruct::init(SpdmVersion::SpdmVersion11, 10),
```

Comment:
why change 100 to 10?

Reply:
This is relative to sending update version (1.3._2_) topics. Will track it with another issue. Removed relative change in this pr.